apps/examples/timer/timer_main.c

```diff
@@ -146,12 +155,41 @@ static pthread_addr_t timer_thread(pthread_addr_t arg)
 		goto error;
 	}
 
+	/* Open the FRTimer device */
+	if (pargs->devno == 0) {
+		frt_dev = 1;
+	} else {
+		frt_dev = 0;
+	}
+	sprintf(path, TIMER_DEVNAME, frt_dev);
+	fprintf(stdout, "FRT Open %s\n", path);
+
+	frt_fd = open(path, O_RDONLY);
```

Comment:
I thinks that frt_fd should close.

Reply:
Done.

Reply:
Better to validate frt_fd before it gets used with ioctl

Reply:
Done. Thanks.